extensions/amp-analytics/0.1/iframe-transport.js

```diff
@@ -138,6 +150,41 @@ export class IframeTransport {
     return frameData;
   }
 
+  createLongTaskObserver_() {
```

Comment:
Could you add a little more documentation for this function?

Reply:
Can do.